fullstop/src/main/java/org/zalando/stups/fullstop/Fullstop.java

```diff
@@ -29,7 +32,9 @@
 /**
  * @author jbellmann
  */
-@SpringBootApplication
+@Configuration
+@ComponentScan
+@EnableAutoConfiguration
```

Comment:
Why did you changed this?  '@SpringBootApplication' is a replacement for the 3 following annotations? I would agree if you try to exclude some 'autoconfigurations', but you don't.